drivers/gles2/rasterizer_storage_gles2.cpp

```diff
@@ -1482,8 +1482,9 @@ void RasterizerStorageGLES2::_update_shader(Shader *p_shader) const {
 	}
 
 	Error err = shaders.compiler.compile(p_shader->mode, p_shader->code, actions, p_shader->path, gen_code);
-
-	ERR_FAIL_COND(err != OK);
+	if (err != OK) {
+		return;
+	}
```

Comment:
In this case `compile` will still properly report the build error before this silent return, right?
i.e. this:
```
1 shader_type spatial;
2 
3 void fragment() {
4       COLOR.x = 
5 }
SHADER ERROR: (null): Expected expression, found: PARENTHESIS_OPEN
   At: :5.
ERROR: _update_shader: Condition ' err != OK ' is true.
   At: drivers/gles2/rasterizer_storage_gles2.cpp:1486.
```
would become this?
```
1 shader_type spatial;
2 
3 void fragment() {
4       COLOR.x = 
5 }
SHADER ERROR: (null): Expected expression, found: PARENTHESIS_OPEN
   At: :5.
```

Reply:
Yes